web/client/components/style/ThemaClassesEditor.jsx

```diff
@@ -129,6 +153,38 @@ class ThemaClassesEditor extends React.Component {
         }
     };
 
+    updateClassification = (classIndex, type) => {
+        let updateIndex;
+        let updateMinMax;
+        let deleteCount = 0;
+        let newClassification = this.props.classification;
+        const currentRule = newClassification[classIndex];
+
+        if (type === 'before') {
+            const isFirstIndex = classIndex === 0;
+            updateIndex = isFirstIndex ? 0 : classIndex;
+            updateMinMax = { min: isFirstIndex ? isNil(currentRule.min) ? currentRule.min : 0 : currentRule.min,
+                max: currentRule.min };
+        } else if (type === 'after') {
+            updateIndex = classIndex === newClassification.length - 1 ? newClassification.length : classIndex + 1;
+            updateMinMax = { min: currentRule.max, max: currentRule.max };
+        } else {
+            updateIndex = classIndex;
+            deleteCount = 1;
+        }
+        let args = [updateIndex, deleteCount];
+        if (type !== 'remove') {
+            const color = '#ffffff';
+            const classifyObj = !isNil(currentRule.unique)
+                ? { ...currentRule, color, title: isNumber(currentRule.unique) ? 0 : '', unique: 0 }
```

Comment:
I think it's missing the conditional operatory on unique property
```suggestion
                ? { ...currentRule, color, title: isNumber(currentRule.unique) ? 0 : '', unique: isNumber(currentRule.unique) ? 0 : '' }
```

maybe it could be written better than what I did